pkg/postgres/schema/groups.go

```diff
@@ -30,11 +33,16 @@ var (
 )
 
 const (
+	// GroupsTableName specifies the name of the table in postgres.
 	GroupsTableName = "groups"
 )
 
 // Groups holds the Gorm model for Postgres table `groups`.
 type Groups struct {
-	PropsId    string `gorm:"column:props_id;type:varchar;primaryKey"`
-	Serialized []byte `gorm:"column:serialized;type:bytea"`
+	PropsID             string `gorm:"column:props_id;type:varchar;primaryKey"`
```

Comment:
this is bordering on the case where having a composite primary key vs using PropsId would be an option.  That said the way you have it more closely mimics how we handle most scenarios and keeps it simpler for maintenance.  Just wanted to point out that the ID we generate is not necessarily providing a ton of value.

Reply:
Yeah, agreed (this was also how it was done beforehand, modulo that the role name was not part of the composite key). We decided to add the ID because this is. what we usually do I suppose.

It definitely has room for improvement in the future.